x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/ssl/SSLReloadDuringStartupIntegTests.java

```diff
@@ -28,6 +29,11 @@
 @ClusterScope(transportClientRatio = 0)
 public class SSLReloadDuringStartupIntegTests extends SecurityIntegTestCase {
 
+    @BeforeClass
+    public static void skipInFips() {
+        assumeFalse("Can't use JKS keystores in FIPS JVM", inFipsJvm());
```

Comment:
Is it possible to make this work if `p12` file is used?

Reply:
Maybe? But we [don't support PKCS12 nor JKS stores in FIPS mode](https://www.elastic.co/guide/en/elasticsearch/reference/current/fips-140-compliance.html#_tls_keystores_and_keys) so there is not much value in testing reloading such a keystore.

Reply:
You are right. Forgot about this again.